cirq-google/cirq_google/engine/abstract_processor.py

```diff
@@ -64,6 +64,8 @@ async def run_async(
         program_labels: Optional[Dict[str, str]] = None,
         job_description: Optional[str] = None,
         job_labels: Optional[Dict[str, str]] = None,
+        run_name: str = "",
```

Comment:
I followed the decision we made in EngineClient to keep it as a String field instead of Optional[String] because there is no semantic difference between "" and None.

However, I do think it would be good to make them all Optional to have more consistency. I can do it in a different PR if that is ok.